Sources/SwiftInspectorCommands/InitializerCommand.swift

```diff
@@ -0,0 +1,84 @@
+// Created by Francisco Diaz on 3/27/20.
+//
+// Copyright (c) 2020 Francisco Diaz
+//
+// Distributed under the MIT License
+//
+// Permission is hereby granted, free of charge, to any person obtaining a copy
+// of this software and associated documentation files (the "Software"), to deal
+// in the Software without restriction, including without limitation the rights
+// to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+// copies of the Software, and to permit persons to whom the Software is
+// furnished to do so, subject to the following conditions:
+//
+// The above copyright notice and this permission notice shall be included in all
+// copies or substantial portions of the Software.
+//
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+// IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+// AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+// LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+// OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+// SOFTWARE.
+
+import ArgumentParser
+import Foundation
+import SwiftInspectorCore
+
+final class InitializerCommand: ParsableCommand {
+  static var configuration = CommandConfiguration(
+    commandName: "initializer",
+    abstract: "Finds information about the initializers of the specified type"
+  )
+
+  @Option(help: "The absolute path to the file to inspect")
+  var path: String
+
+  @Option(help: "The name of the type to look information of the initializer")
+  var name: String
+
+  @Flag(name: .shortAndLong, default: true, inversion: .prefixedEnableDisable, help: Help.typeOnly)
+  var typeOnly: Bool
+
+  /// Runs the command
+  func run() throws {
+    let cachedSyntaxTree = CachedSyntaxTree()
+    let analyzer = InitializerAnalyzer(name: name, cachedSyntaxTree: cachedSyntaxTree)
+    let fileURL = URL(fileURLWithPath: path)
+
+    let initializerStatements = try analyzer.analyze(fileURL: fileURL)
+    let outputArray = initializerStatements.map { outputString(from: $0)}
+
+    let output = outputArray.joined(separator: "\n")
```

Comment:
I think it's nice when it prints nothing when it doesn't find anything. Can we do something like this? https://github.com/fdiaz/SwiftInspector/blob/0ac2841c36483d2097e9ed7a15db9a235d9c2bcf/Sources/SwiftInspectorCommands/TypeLocationCommand.swift#L49-L51

Reply:
As is I think it prints a newline.

Reply:
I haven't consider this. Do you  think it's problematic to print a newline? I'd prefer to keep the code simple if we can but if this might become troublesome I think we should remove it.

Reply:
I don't think it's terrible but it's not idiomatic. Sibling CLI tools that find stuff (`grep`, `sed`, `find`) are all silent when they don't find anything. I think there is some value in following their lead.